src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -123,13 +123,13 @@ public TypeSymbolWithAnnotations InferredReturnType(ref HashSet<DiagnosticInfo>
             out RefKind refKind,
             out bool inferredFromSingleType)
         {
+            var conversions = binder.Conversions;
```

Comment:
It seems the local `conversions` is only used in one place. Consider inlining.

Reply:
Done.